examples/cifar10_cnn_capsule.py

```diff
@@ -1,59 +1,51 @@
 """
-#Train a simple CNN-Capsule Network on the CIFAR10 small images dataset.
+This example trains a simple CNN-Capsule Network on the CIFAR10 data set.
 
 Without Data Augmentation:
-It gets to 75% validation accuracy in 10 epochs,
-and 79% after 15 epochs, and overfitting after 20 epochs
+It gets to 75% validation accuracy in 10 epochs, 79% after 15 epochs,
+and overfitting after 20 epochs
 
 With Data Augmentation:
-It gets to 75% validation accuracy in 10 epochs,
-and 79% after 15 epochs, and 83% after 30 epochs.
-In my test, highest validation accuracy is 83.79% after 50 epochs.
+It gets to 75% validation accuracy in 10 epochs, 79% after 15 epochs,
+and 83% after 30 epochs.
 
-This is a fast Implement, just 20s/epoch with a gtx 1070 gpu.
-"""
+The highest achieved validation accuracy is 83.79% after 50 epochs.
 
+This is a fast implementation that takes just 20s/epoch on a GTX 1070 GPU.
+"""
 from __future__ import print_function
-from keras import backend as K
-from keras.layers import Layer
 from keras import activations
+from keras import backend as K
 from keras import utils
 from keras.datasets import cifar10
-from keras.models import Model
-from keras.layers import *
+from keras.layers import AveragePooling2D, Conv2D, Lambda, Layer, Reshape
+from keras.models import Input, Model
 from keras.preprocessing.image import ImageDataGenerator
 
 
-# the squashing function.
-# we use 0.5 in stead of 1 in hinton's paper.
-# if 1, the norm of vector will be zoomed out.
-# if 0.5, the norm will be zoomed in while original norm is less than 0.5
-# and be zoomed out while original norm is greater than 0.5.
+# The squashing function.
+# 0.5 is used instead of 1 as in the paper.
+# If 1, the norm of vector will be zoomed out.
+# If 0.5, the norm will be zoomed in.
 def squash(x, axis=-1):
     s_squared_norm = K.sum(K.square(x), axis, keepdims=True) + K.epsilon()
     scale = K.sqrt(s_squared_norm) / (0.5 + s_squared_norm)
     return scale * x
 
 
-# define our own softmax function instead of K.softmax
-# because K.softmax can not specify axis.
-def softmax(x, axis=-1):
-    ex = K.exp(x - K.max(x, axis=axis, keepdims=True))
-    return ex / K.sum(ex, axis=axis, keepdims=True)
-
-
-# define the margin loss like hinge loss
+# Define the margin loss like the hinge loss
 def margin_loss(y_true, y_pred):
     lamb, margin = 0.5, 0.1
     return K.sum(y_true * K.square(K.relu(1 - margin - y_pred)) + lamb * (
         1 - y_true) * K.square(K.relu(y_pred - margin)), axis=-1)
 
 
 class Capsule(Layer):
-    """A Capsule Implement with Pure Keras
-    There are two vesions of Capsule.
-    One is like dense layer (for the fixed-shape input),
-    and the other is like timedistributed dense (for various length input).
+    """A Capsule Network implementation in Keras
```

Comment:
Please standardize the docstrings formatting to be the same as other docstrings in the codebase.